client/my-sites/domains/style.scss

```diff
@@ -19,22 +19,6 @@
 		margin-bottom: auto;
 	}
 
-	.popover-cart {
```

Comment:
This still occurs in a e2e test suite, so it provides an opportunity for further cleanup.

Reply:
Same comment as above.